data/battle_scripts_1.s

```diff
@@ -5756,11 +5757,12 @@ BattleScript_PursuitDmgOnSwitchOut::
 	resultmessage
 	waitmessage B_WAIT_TIME_LONG
 	tryfaintmon BS_TARGET
-	moveendfromto MOVEEND_ABILITIES, MOVEEND_CHOICE_MOVE
+	moveendfromto MOVEEND_ABILITIES, MOVEEND_CHOICE_MOVE+1
```

Comment:
This feels a bit hacky. If we just need to add one, why not use the actual define for it (`MOVEEND_ATTACKER_INVISIBLE`)?

Reply:
Sure, I can change it to MOVEEND_ATTACKER_INVISIBLE

Reply:
Done